src/tests/tests/lists.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+  QUnit.module('Test lists');
+  let mdfp = require('multiDomainFP');
+
+  QUnit.test('Check MDFP domains are all base domains', (assert) => {
+    for (var group of mdfp.multiDomainFirstPartiesArray) {
+      for (var domain of group) {
+        assert.ok(window.getBaseDomain(domain) == domain, 'is a basedomain');
```

Comment:
Would be nice to specify the domain being tested in the assertion message, so that if any domains fail, it's immediately clear which.

Reply:
done